src/containers/App/App.js

```diff
@@ -158,20 +127,18 @@ App.propTypes = {
   }).isRequired,
 };
 
-export default withTranslation()(
-  connect(
-    state => ({
-      language: state.app.language,
-    }),
-    dispatch => ({
-      actions: bindActionCreators(
-        {
-          updateViewportDimensions,
-          onScroll,
-          updateLanguage,
-        },
-        dispatch
-      ),
-    })
-  )(App)
-);
+export default connect(
```

Comment:
How are you able to export this without a function wrapping it? (Do other files have to specify their own name for it when importing?)

Reply:
(Or am I misunderstanding what `withTranslation` was doing?)